SignalServiceKit/src/Network/OWSSignalService.m

```diff
@@ -141,9 +141,9 @@ - (void)setIsCensorshipCircumventionActive:(BOOL)isCensorshipCircumventionActive
     }
 
     [[NSNotificationCenter defaultCenter]
-        postNotificationName:kNSNotificationName_IsCensorshipCircumventionActiveDidChange
-                      object:nil
-                    userInfo:nil];
+        postNotificationNameAsync:kNSNotificationName_IsCensorshipCircumventionActiveDidChange
+                           object:nil
+                         userInfo:nil];
```

Comment:
Observed by navigation view and socket manager. Posting later safe in both cases.